ui/raidboss/data/07-dt/raid/r1s.ts

```diff
@@ -100,141 +65,15 @@ const mapEffectData = {
     'centerY': 105,
     ...mapEffectTileState,
   },
-  '0B': {
-    'location': '0B',
-    'centerX': 115,
-    'centerY': 105,
-    ...mapEffectTileState,
-  },
-  '0C': {
-    'location': '0C',
-    'centerX': 85,
-    'centerY': 115,
-    ...mapEffectTileState,
-  },
-  '0D': {
-    'location': '0D',
-    'centerX': 95,
-    'centerY': 115,
-    ...mapEffectTileState,
-  },
-  '0E': {
-    'location': '0E',
-    'centerX': 105,
-    'centerY': 115,
-    ...mapEffectTileState,
-  },
-  '0F': {
-    'location': '0F',
-    'centerX': 115,
-    'centerY': 115,
-    ...mapEffectTileState,
-  },
-  '10': {
-    'location': '10',
-    'centerX': 85,
-    'centerY': 85,
-    ...mapEffectTileOverlay,
-  },
-  '11': {
-    'location': '11',
-    'centerX': 95,
-    'centerY': 85,
-    ...mapEffectTileOverlay,
-  },
-  '12': {
-    'location': '12',
-    'centerX': 105,
-    'centerY': 85,
-    ...mapEffectTileOverlay,
-  },
-  '13': {
-    'location': '13',
-    'centerX': 115,
-    'centerY': 85,
-    ...mapEffectTileOverlay,
-  },
-  '14': {
-    'location': '14',
-    'centerX': 85,
-    'centerY': 95,
-    ...mapEffectTileOverlay,
-  },
-  '15': {
-    'location': '15',
-    'centerX': 95,
-    'centerY': 95,
-    ...mapEffectTileOverlay,
-  },
-  '16': {
-    'location': '16',
-    'centerX': 105,
-    'centerY': 95,
-    ...mapEffectTileOverlay,
-  },
-  '17': {
-    'location': '17',
-    'centerX': 115,
-    'centerY': 95,
-    ...mapEffectTileOverlay,
-  },
-  '18': {
-    'location': '18',
-    'centerX': 85,
-    'centerY': 105,
-    ...mapEffectTileOverlay,
-  },
-  '19': {
-    'location': '19',
-    'centerX': 95,
-    'centerY': 105,
-    ...mapEffectTileOverlay,
-  },
-  '1A': {
-    'location': '1A',
-    'centerX': 105,
-    'centerY': 105,
-    ...mapEffectTileOverlay,
-  },
-  '1B': {
-    'location': '1B',
-    'centerX': 115,
-    'centerY': 105,
-    ...mapEffectTileOverlay,
-  },
-  '1C': {
-    'location': '1C',
-    'centerX': 85,
-    'centerY': 115,
-    ...mapEffectTileOverlay,
-  },
-  '1D': {
-    'location': '1D',
-    'centerX': 95,
-    'centerY': 115,
-    ...mapEffectTileOverlay,
-  },
-  '1E': {
-    'location': '1E',
-    'centerX': 105,
-    'centerY': 115,
-    ...mapEffectTileOverlay,
-  },
-  '1F': {
-    'location': '1F',
-    'centerX': 115,
-    'centerY': 115,
-    ...mapEffectTileOverlay,
-  },
 } as const;
 
 const headMarkerData = {
   // Vfx Path: tank_lockon02k1
   tankbuster: '00DA',
   // Vfx Path: lockon8_t0w
-  lineStack: '00F4',
+  spreadMarker1: '00F4',
```

Comment:
I would have called this "nailchipper" as thats the name of the attack that gives the marker, but spread is probably fine.